src/quo/components/animated/pressable.cljs

```diff
@@ -69,26 +70,29 @@
          :or                 {border-radius 0
                               type          "primary"}}
         (bean/bean props)
-        long-press-ref       (react/create-ref)
-        state                (animated/use-value (:undetermined gesture-handler/states))
-        active               (animated/eq state (:began gesture-handler/states))
-        gesture-handler      (animated/use-gesture {:state state})
-        animation            (react/use-memo
-                              (fn []
-                                (animated/with-timing-transition active
-                                  {:duration (animated/cond* active time-in time-out)
-                                   :easing   (:ease-in animated/easings)}))
-                              [])
+        long-press-ref (react/create-ref)
+        state (animated/use-value (:undetermined gesture-handler/states))
+        active (animated/eq state (:began gesture-handler/states))
+        gesture-handler (animated/use-gesture {:state state})
+        animation (react/use-memo
+                   (fn []
+                     (animated/with-timing-transition active
+                                                      {:duration (animated/cond* active time-in time-out)
+                                                       :easing   (:ease-in animated/easings)}))
+                   [])
         {:keys [background
-                foreground]} (react/use-memo
-                              (fn []
-                                (type->animation {:type      (keyword type)
-                                                  :animation animation}))
-                              [type])
-        handle-press         (fn [] (when on-press (on-press)))
+                foreground]}
+        (react/use-memo
+         (fn []
+           (type->animation {:type      (keyword type)
+                             :animation animation}))
+         [type])
+        handle-press (fn [] (when on-press (on-press)))
         long-gesture-handler (react/callback
                               (fn [^js evt]
-                                (let [gesture-state (-> evt .-nativeEvent .-state)]
+                                (let [gesture-state (-> evt
+                                                        .-nativeEvent
+                                                        .-state)]
```

Comment:
looks like unnecessary change to me

Reply:
[Personal guess] I'd also prefer if it didn't add line breaks. Perhaps it's related to the max line length, then zprint applies the Clojure Style Guide rule to break arguments in their own line.

Reply:
Checked the code. This is not caused by `:width`. Still looking for ways to customize this, the most straightforward way to custom `->` won't work.